airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml

```diff
@@ -52,6 +52,8 @@ acceptance_tests:
               bypass_reason: "Data changing frequently, because we use live account."
             - name: position
               bypass_reason: "Data changing frequently, because we use live account."
+            - name: country
+              bypass_reason: "Data changing frequently, because we use live account."
```

Comment:
Do we plan to not use a live account? There are a lot of ignored fields that drastically lower the coverage of the tests. cc @YowanR

Reply:
I think in future we should do that. Already inform @YowanR about that.